osu.Game/Online/Chat/ExternalLinkOpener.cs

```diff
@@ -44,26 +45,26 @@ public partial class ExternalLinkDialog : PopupDialog
         {
             public ExternalLinkDialog(string url, Action openExternalLinkAction, Action copyExternalLinkAction)
             {
-                HeaderText = "Just checking...";
-                BodyText = $"You are about to leave osu! and open the following link in a web browser:\n\n{url}";
+                HeaderText = "You are leaving osu!";
+                BodyText = $"Are you sure you want to open the following link in a web browser:\n\n{url}";
```

Comment:
This is worded as a question now:

```suggestion
                BodyText = $"Are you sure you want to open the following link in a web browser?\n\n{url}";
```